decidim-admin/app/forms/decidim/admin/participatory_space_private_user_csv_import_form.rb

```diff
@@ -1,13 +1,27 @@
 # frozen_string_literal: true
 
+require "csv"
+
 module Decidim
   module Admin
     # A form object used to upload CSV to batch participatory space private users.
     #
     class ParticipatorySpacePrivateUserCsvImportForm < Form
       attribute :file
+      attribute :name, String
+      attribute :email, String
 
       validates :file, presence: true
+      validate :validate_csv
+
+      def validate_csv
+        if file.present?
+          CSV.foreach(file.path) do |email, user_name|
+            errors.add(:user_name, "user_name not valid!") unless user_name.match?(/\A(?!.*[<>?%&\^*#@\(\)\[\]\=\+\:\;\"\{\}\\\|])/)
```

Comment:
The error message should be localized please

Reply:
Fixed not localized error message :smile: